internal/hcsoci/resources.go

```diff
@@ -76,7 +76,7 @@ type scsiMount struct {
 }
 
 // TODO: Method on the resources?
-func ReleaseResources(ctx context.Context, r *Resources, vm *uvm.UtilityVM, all bool) error {
+func ReleaseResources(ctx context.Context, r *Resources, vm *uvm.UtilityVM, id string, all bool) error {
```

Comment:
Can we just add `id` to the `Resources` struct. Seems better anyways and then we can associate resources to the appropriate container anyways.

Reply:
Done.